### ISSUE TITLE
Question: Candidate Gathering

### ISSUE THREAD

**Issue by 4ydx** [pixelbender/go-stun#2]
How difficult would you consider adding this feature to be?

**Comment by pixelbender** [comment]
@4ydx Yes, exactly, ICE may use candidates discovered by STUN (server reflexive) and TURN (relayed) candidates.Right now I am working on multiplexed STUN transport. It allows usage of ICE or other protocols (such as RTP, RTCP, DTLS) over the same net.Conn. Gathering ICE candidates is pretty simple. But there are other procedures to implement: prioritizing, selecting pairs, restarting, etc.
